src/ape_run/_cli.py

```diff
@@ -53,7 +54,13 @@ def _get_command(self, filepath: Path) -> Union[click.Command, click.Group, None
 
             with use_temp_sys_path(filepath.parent.parent):
                 try:
-                    ns = run_module(f"scripts.{filepath.stem}")
+                    path = (
+                        str(filepath)
+                        .replace(os.path.sep, ".")
+                        .split("scripts.")[-1]
+                        .replace(filepath.suffix, "")
+                    )
+                    ns = run_module(f"scripts.{path}")
                 except Exception as e:
                     logger.error_from_exception(
                         e, f"Exception while parsing script: {relative_filepath}"
```

Comment:
To fix the middle-dir click scripts, checnage the part a couple lines below from:

```python
return ns["cli"]
```

to 

```python
cli_obj = ns["cli"]
cli_obj.name = filepath.stem
return cli_obj
```

Reply:
perhaps we should only set the name if it not equal to `"cli"` though.

Reply:
ok awesome this got it working